laravel/routes/web.php

```diff
@@ -20,12 +21,17 @@
 //topページ
 Route::resource('top',TopController::class)->only([
     'index'
-]);;
+]);
 
 //投稿関連
 Route::resource('posts',PostController::class)->only([
     'create', 'store','destroy','edit','update'
-]);;
+]);
+
+//コメント機能
+Route::resource('/posts/{post}/comments',ReplieController::class)->only([
```

Comment:
post_idのほうがpostsテーブルのidが渡ってくるということが明確になるのでおすすめです
```suggestion
Route::resource('/posts/{post_id}/comments',ReplieController::class)->only([
```

Reply:
comments部も表記かえました！
Route::resource('/posts/{post_id}/Replies',ReplieController::class)->only([
    'index','store'
]);